l10n_ch_invoice_with_payment/readme/HISTORY.rst

```diff
@@ -0,0 +1,5 @@
+11.0.1.0.0 (2019-06-07)
+~~~~~~~~~~~~~~~~~~~~~~~
+
+* [ADD] New module
```

Comment:
Maybe you want to add a new entry in this file for migration in version 12.0?

Reply:
I prefer to rid of this file as no one will write the changes here

Reply:
updated, thanks